File: Data Structures & Algorithms/duplicate-integer/submission-1.cpp
class Solution {
public:
    bool hasDuplicate(vector<int>& nums) {
        
        for(int i=0;i<nums.size();i++){
            
            int counts=count(nums.begin(), nums.end(), nums[i]);
            if(counts>1) return true;    
        }
        return false;
    }
};

File: Data Structures & Algorithms/top-k-elements-in-list/submission-1.py
class Solution:
    def topKFrequent(self, nums: List[int], k: int) -> List[int]:
        sets=set(nums)
        hashm:dict[int,int]={}
        lists=[]
        count1=0
        c=0
        for i in sets:
            count=0
            for j in nums:
                if(i==j):
                    count+=1;
            hashm.update({i:count})    

        for i in range(k):
            c=max(hashm,key=hashm.get)
            lists.append(c)
            del(hashm[c])

        return lists

            
